### ISSUE TITLE
Server warns: "Getopt::Long::Descriptive was configured with these ambiguous options: h"

### ISSUE THREAD

**Issue by dboehmer** [perl-catalyst/catalyst-runtime#179]
Since some time starting the `server.pl` of any Catalyst app generates the following warning:

```console
$ catalyst.pl MyApp
...
$ cd MyApp/
$ script/myapp_server.pl -dr
Getopt::Long::Descriptive was configured with these ambiguous options: h
[debug] Debug messages enabled
[debug] Statistics enabled
...
```

**Comment by haarg** [comment]
Do you have the latest version of Catalyst installed? This was fixed in v5.90129.

**Comment by dboehmer** [comment]
Thanks for the fast reply and sorry, I should have checked that.
